lime/lime_tabular.py

```diff
@@ -189,6 +191,7 @@ def __init__(self,
         self.categorical_names = categorical_names or {}
         self.sample_around_instance = sample_around_instance
         self.training_data_stats = training_data_stats
+        self.model = None #to access model after the fact
```

Comment:
Do we need this?

Reply:
I can remove that, I was using this previously to investigate things as I was working on understanding the LIME code.